### PR TITLE
Steve dev

### PR DESCRIPTION
**Pull request by mbouadeus**
Added a few features and got portfolio ready for deployment.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Steve!</h1>");
+    response.getWriter().println("If you can dream it, you can do it!");
   }
 }
```
Comment (pengxianzhe) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: no newline at EOF

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -22,18 +22,45 @@ window.addEventListener('load', function () {
...
+function setBackground() {
+    // Retrieves id of background option selected in menu.
+    const id = getSelectedBackgroundId();
+
+    // Sets background
+    addBackground(id);
+
+    // Closes background selection menu.
+    $('#settingsModal').modal('hide');
+}
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/script.js]: nit: no new line at EOF

File: portfolio/src/main/webapp/test.html [hunk 3]
```diff
@@ -2,22 +2,36 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Contact | My Portfolio</title>
+    <title>My Portfolio</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" 
         integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="test.js"></script>
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/test.html]: Let's change the name of this file. test.js isn't a very informative name. What are you trying to do with test.js?

File: portfolio/src/main/webapp/test.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Retreives quote from server.
+ */
+
+ function getQuote() {
+     fetch('/data').then(response => response.text()).then(quote => {
+         document.getElementById('quote-container').innerText = quote;
+     });
+ }
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/test.js]: nit: no newline at EOF.

**Review by pengxianzhe — CHANGES_REQUESTED** [review]
One thing I noticed (and should notice earlier) is that you are using 4 spaces for indentation. In Google the standard is 2 spaces. Do you mind to change it to 2 spaces? There is an option in settings.

File: portfolio/src/main/webapp/ServerSideTrainings.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      </div>
+    </main>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" 
+      integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" 
+      integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" 
+      integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+  </body>
+</html>
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/ServerSideTrainings.html:56]: nit: no newline at EOF.

File: portfolio/src/main/webapp/about.html [hunk 6]
```diff
@@ -4,95 +4,95 @@
...
+      integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" 
-        integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+      integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" 
-        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+      integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
   </body>
-</html>
+</html>
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/about.html:98]: nit: no newline at EOF.

File: portfolio/src/main/webapp/blog.html [hunk 7]
```diff
@@ -4,85 +4,85 @@
...
+      integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" 
-        integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+      integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" 
-        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+      integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
   </body>
-</html>
+</html>
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/blog.html:88]: nit: no newline at EOF.

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -4,79 +4,81 @@
...
-        integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+      integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" 
-        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+      integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
   </body>
-  
-</html>
-
+</html>
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/index.html:84]: nit: no newline at EOF.

**Review by pengxianzhe — APPROVED** [review]
The changes LGTM so I am approving it. You can go ahead and merge once you are done with the newlines

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (mbouadeus) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Fixed

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (mbouadeus) [reply portfolio/src/main/webapp/script.js]: Fixed.

File: portfolio/src/main/webapp/test.html (hunk 3 above)
Comment (mbouadeus) [reply portfolio/src/main/webapp/test.html]: Fixed. Will try using more descriptive names in future.

File: portfolio/src/main/webapp/test.js (hunk 4 above)
Comment (mbouadeus) [reply portfolio/src/main/webapp/test.js]: Fixed.

File: portfolio/src/main/webapp/ServerSideTrainings.html (hunk 5 above)
Comment (mbouadeus) [reply portfolio/src/main/webapp/ServerSideTrainings.html:56]: Fixed.

**Pull request merged**